docker-entrypoint.sh

```diff
@@ -35,6 +35,17 @@ write_env_to_json() {
 write_env_to_json /usr/share/nginx/html/env.json
 
 
+# TODO remove when FHIR resources are PUT after SoF launch
+if [ -n "$FHIR_SERVER" ]; then
+    echo Uploading files:
+    find /var/opt
```

Comment:
debugging cruft?

Reply:
I thought it might be helpful, in case someone added a file and didn't see it uploaded. Happy to remove either way